apps/src/templates/sectionAssessments/MultipleChoiceByStudentSection.jsx

```diff
@@ -0,0 +1,41 @@
+import React, {Component, PropTypes} from 'react';
+import StudentAssessmentOverviewTable from './StudentAssessmentOverviewTable';
+import { studentAnswerDataPropType, questionStructurePropType } from './assessmentDataShapes';
+import {
+  getMultipleChoiceStructureForCurrentAssessment,
+  getStudentMCResponsesForCurrentAssessment,
+} from './sectionAssessmentsRedux';
+import { connect } from 'react-redux';
+
+class MultipleChoiceByStudentSection extends Component {
```

Comment:
What is a "student section"?  Do you mean "by student by section" in this name?

Reply:
MultipleChoiceByStudent, section (aka container). Maybe it's helpful to call it container and add a comment.